### PR TITLE
Add the tic-tac-toe application

### PR DESCRIPTION
**Pull request by Jacalz**
I have tried to get this as working as possible, but I could not find any information on how to test this myself. We talked about this a long time ago and now it has finally happened :) @andydotxyz

### CODE REVIEW

File: _apps/tic-tac-toe.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+icon: https://github.com/jacalz/go-projects/blob/master/school/tic-tac-toe/assets/icon256.png?raw=true
+screenshot1: https://github.com/jacalz/go-projects/blob/master/school/tic-tac-toe/assets/tic-tac-toe.png?raw=true
+
+date:      2020-02-19 14:42:00
+excerpt:   A simple tic-tac-toe game built using Fyne.
+category:  game
+developer: Jacalz
+
+git: https://github.com/Jacalz/go-projects.git
+package: github.com/Jacalz/go-projects/tree/master/school/tic-tac-toe
```
Comment (andydotxyz) [inline _apps/tic-tac-toe.md]: This line isn't right - it should be the go package used in "go get" to install the app. Probably just removing "tree/master" should work.

**Comment by andydotxyz** [comment]
Nice addition, thanks. To test use "jekyll serve" but it depends on some ruby tools etc - probably useful docs on https://jekyllrb.com/docs/installation/

File: _apps/tic-tac-toe.md (hunk 1 above)
Comment (Jacalz) [reply _apps/tic-tac-toe.md]: Yeah, good catch. Copy paste issues are the worst...

**Comment by Jacalz** [comment]
The package import path should be fixed now. I tried my hardest to get jekyll to work, but I just could not get it working at all, I am sorry.

**Comment by Jacalz** [comment]
Just have two smaller changes that I want to do. hang tight.

**Pull request merged**